pkg/util/workloadrepo/snapshot.go

```diff
@@ -115,6 +116,10 @@ func (w *worker) snapshotTable(ctx context.Context, snapID uint64, rt *repositor
 	defer w.sesspool.Put(_sessctx)
 	sess := _sessctx.(sessionctx.Context)
 
+	if rt.test && intest.InTest {
```

Comment:
IMHO `test` is quite confusing as a field of `repositoryTable`. Also, potentially this makes `snapshotTable` disabled for even 'real tikv tests'.

How about:

1. Renaming `repositoryTable.test` to something like `repositoryTable.requireRealTiKV`(or even better, add a variable to the table definition in `pkg/infoschema/tables.go` since this is an attribute of the table).
2. Changing `intest.InTest` to `usingRealTiKV`(like the [code here]( https://github.com/pingcap/tidb/blob/5ac0b2e033bda6dbf30aa7d1989f5841b6931060/pkg/executor/infoschema_reader.go#L1978-L1980))